index.html

```diff
@@ -40,6 +42,188 @@ <h4>I’am a Software Developer</h4>
             <li><a href="#"><img src="images/5.png" alt="M_logo"></a></li>
         </ul>
     </section>
+ <!-- Recent Work Start -->
+ <section class="recent-work">
+    <div class="title">
+        <h1 class="h-title">My Recent Works</h1>
+        <hr>
+    </div>
+    <div class="recent-frame-1">
+        <div><img src="/images/img-placeholder-1.png" alt=""></div>
+        <h1 class="h-title">Multi-Post Stories</h1>
+        <p class="post-description">A daily selection of privately personalized reads; no accounts or sign-ups
+            required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took
+            a standard dummy text.</p>
+        <ul class="post-tags">
+            <li>css</li>
+            <li>html</li>
+            <li>bootstrap</li>
+            <li>Ruby</li>
+        </ul>
+        <button class="post-btn-enable" type="button">See Project</button>
+    </div>
+    <div class="recent-frame-2">
+        <div class="card-work">
+            <div class="post-right-block">
+                <h1 class="card-post-title">Profesional Art Printing Data</h1>
+                <p class="supporting-text">A daily selection of privately personalized reads; no accounts or
+                    sign-ups required. has been
+                    the industry's standard</p>
+                <ul class="post-tags-2">
+                    <li>html</li>
+                    <li>bootstrap</li>
+                    <li>Ruby</li>
+                </ul>
+            </div>
+            <div class="btn-action">
+                <div class="post-btn-enable-2">
+                    <h1 class="btn-text">See Project</h1>
```

Comment:
- [ ] Nice Working on your `Works Section`, kindly make the `see projects`  a link `a` or `button` tag, for better use of HTML semantics.